### DISCUSSION TITLE
OMI-Champions Meeting 2022-07-29

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#260] [General]
- we moved up our meeting a week 
- ht: we created an ecosystem where people seem to be afraid to admit when we said we would do something but we couldn't because of something coming up. 
    - lets flip it around that its not only acceptable but also practical. Just call it out. 
    - what is the process for passing the torch reliably? 
- submit PR to remove OMi-chan from readme for now 
- archive interoperator, as we believe MSF-delegates and Champions are satisfying those 
- remove interoperator and website group meetings from calendar (and readme)
- exploring removing dependency on w3c emphasis. 
- maybe we decouple the technical w3c slot 
- organization co-chairs for other things non-w3c
- propose official recognition of m3's avatar group (and maybe add a link to the readme to that group/repo? this hasn't been done before, but its something we should explore)
- working group leads: carve out the last 5 min of your meetings to "create the agenda for next week, add the things we didn't get to finish today, and you all can add anything to the agenda before we meet again, here is the link"
- what are the minimum expectations of working group? (we have a ticket for this) 
- how do we fail fast, instead of having defunct groups suffer along on fumes? How can we normalize calling out when things aren't working?

**Comment by mrmetaverse** [comment]
Created https://github.com/omigroup/omigroup/issues/261

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/omi-chan/discussions/4#discussioncomment-3283773 Jin proposed a solution related to avatar x omi-chan sunsetting

**Comment by mrmetaverse** [comment]
unblocked #244

**Comment by humbletim** [comment]
attendees (taken near start of meeting; forgot to take an updated snapshot towards end but P1 RedEagle also attended; does this miss reflecting any others?)
![image](https://user-images.githubusercontent.com/94753/181843975-3c5c4058-6a7b-4492-9b78-3aee61b9ee97.png)

**Comment by mrmetaverse** [comment]
Going to try out a weekly Champion async Standup: 
- What I did This week, What I plan on looking at Next week , known blockers 

Voluntary, and in beta.

**Comment by mrmetaverse** [comment]
Backlog refinement special session 08/02/22
- I may transfer this to a new doc but for now everything I need is here 
- I will use a checklist method for the things I am taking care of during today's session. This means I will update this comment repeatedly. 
- [x] submit PR to remove OMi-chan from readme for now
- [x] archive interoperator, as we believe MSF-delegates and Champions are satisfying those
- [x] remove interoperator and website group meetings from the calendar (and readme)

**Reply by mrmetaverse** [reply]
cleaned up some typos first and freshened up messaging. This led to a mention of the omi champions, which I think is appropriate. Where we used to have just a mention of the chairs.

**Reply by mrmetaverse** [reply]
We looked at https://github.com/omigroup/omi-chan/discussions/4#discussioncomment-3283773 re: recognizing m3 avatar group.

**Reply by mrmetaverse** [reply]
@humbletim submitted [this PR](https://github.com/omigroup/omigroup/tree/756e18558725971209f74aa8ab3ae0c525c69c30) that completely reorganized our readme to a more modern picture of the OMI landscape.

**Reply by mrmetaverse** [reply]
We aren't exactly submitting a PR to remove OMI-chan. It is an inactive group seeking a new lead - specifically, the OMI chan effort is for the OMI mascot and interoperable avatar. 

https://github.com/omigroup/omigroup/issues/137

**Reply by mrmetaverse** [reply]
archived OMI-chan 
![image](https://user-images.githubusercontent.com/63426722/182480842-11b7f93b-b447-4a3e-8b23-1fb032d27af3.png)

**Reply by mrmetaverse** [reply]
archives are ideal because we can un-archive if it is revived.
